### ISSUE TITLE
Build or Generate fails on Windows

### ISSUE THREAD

**Issue by vuchl** [nuxt/content#711]
### Version
@nuxt/content: 1.11.1
nuxt: 2.14.7 
@nuxt/content-theme-docs: 0.8.0

### Steps to reproduce
Running `nm run build` or `npm run generate`on windows fails with an error message:

```cmd
npm run generate

> configurator-docs@1.0.0 generate C:\repos\content-docs
> nuxt generate

i Merging Tailwind config from ~/..\..\..\..\tailwind.config.js                                                  nuxt:tailwindcss 14:45:40
i Parsed 8 files in 1,3 seconds                                                                                     @nuxt/content 14:45:45
i Production build                                                                                                                14:45:48
i Bundling for server and client side                                                                                             14:45:48
i Target: static                                                                                                                  14:45:48
√ Builder initialized                                                                                                             14:45:48
√ Nuxt files generated                                                                                                            14:45:50

√ Client
  Compiled successfully in 50.40s

* Server █████████████████████████ after chunk asset optimization (93%) SourceMapDevToolPlugin components_global/base/Alert.js
 generate SourceMap


 FATAL  EINVAL: invalid argument, mkdir 'C:\repos\content-docs\node_modules\.cache\nuxt\dist\server\C:_repos\content-docs\components\global'

```

### What is Expected?

The build should succeed. 

### What is actually happening?

Propably the path building gets messed up somewhere. Maybe a creoss-env issue?

**Comment by vuchl** [comment]
I can `build`and `generate`in Windows with e.g. an Alpine WSL command prompt.

**Comment by vuchl** [comment]
The directory is existing and also holds a few .vue components. 
![image](https://user-images.githubusercontent.com/4921147/104195822-60d39680-5423-11eb-8c5d-bde420a13ea3.png)

**Comment by vuchl** [comment]
Where can I find the build scripts?

**Comment by vuchl** [comment]
I dug a bit into the CLI and build process. I found it seems to be more related to webpack. I get till here https://github.com/nuxt/nuxt.js/blob/dev/packages/builder/src/builder.js#L166 without problems. Then it seems the build process is handed over to webpack and run in another process. There a wrong path is built and handed to `mkdirp`. A relating issue can be found here: https://github.com/react-static/react-static/issues/1360

**Comment by vuchl** [comment]
Could it be something related to chunk name generation?

Here is my `.nuxt/components/plugins.js`
```js
import Vue from 'vue'

const globalComponents = {
  AppColorSwitcher: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\app\\AppColorSwitcher.vue' /* webpackChunkName: "components_global/app/AppColorSwitcher" */).then(c => c.default || c),
  Alert: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\base\\Alert.vue' /* webpackChunkName: "components_global/base/Alert" */).then(c => c.default || c),
  Badge: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\base\\Badge.vue' /* webpackChunkName: "components_global/base/Badge" */).then(c => c.default || c),
  CodeBlock: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\base\\CodeBlock.vue' /* webpackChunkName: "components_global/base/CodeBlock" */).then(c => c.default || c),
  CodeGroup: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\base\\CodeGroup.vue' /* webpackChunkName: "components_global/base/CodeGroup" */).then(c => c.default || c),
  CodeSandbox: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\base\\CodeSandbox.vue' /* webpackChunkName: "components_global/base/CodeSandbox" */).then(c => c.default || c),
  List: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\base\\List.vue' /* webpackChunkName: "components_global/base/List" */).then(c => c.default || c),
  BuiltWithNuxtDark: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\BuiltWithNuxtDark.vue' /* webpackChunkName: "components_global/icons/BuiltWithNuxtDark" */).then(c => c.default || c),
  BuiltWithNuxtLight: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\BuiltWithNuxtLight.vue' /* webpackChunkName: "components_global/icons/BuiltWithNuxtLight" */).then(c => c.default || c),
  IconArrowLeft: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\IconArrowLeft.vue' /* webpackChunkName: "components_global/icons/IconArrowLeft" */).then(c => c.default || c),
  IconArrowRight: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\IconArrowRight.vue' /* webpackChunkName: "components_global/icons/IconArrowRight" */).then(c => c.default || c),
  IconBadgeCheck: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\IconBadgeCheck.vue' /* webpackChunkName: "components_global/icons/IconBadgeCheck" */).then(c => c.default || c),
  IconCheckCircle: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\IconCheckCircle.vue' /* webpackChunkName: "components_global/icons/IconCheckCircle" */).then(c => c.default || c),
  IconChevronRight: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\IconChevronRight.vue' /* webpackChunkName: "components_global/icons/IconChevronRight" */).then(c => c.default || c),
  IconClipboardCheck: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\IconClipboardCheck.vue' /* webpackChunkName: "components_global/icons/IconClipboardCheck" */).then(c => c.default || c),
  IconClipboardCopy: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\IconClipboardCopy.vue' /* webpackChunkName: "components_global/icons/IconClipboardCopy" */).then(c => c.default || c),
  IconExclamationCircle: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\IconExclamationCircle.vue' /* webpackChunkName: "components_global/icons/IconExclamationCircle" */).then(c => c.default || c),
  IconExternalLink: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\IconExternalLink.vue' /* webpackChunkName: "components_global/icons/IconExternalLink" */).then(c => c.default || c),
  IconGithub: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\IconGithub.vue' /* webpackChunkName: "components_global/icons/IconGithub" */).then(c => c.default || c),
  IconInformationCircle: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\IconInformationCircle.vue' /* webpackChunkName: "components_global/icons/IconInformationCircle" */).then(c => c.default || c),
  IconMenu: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\IconMenu.vue' /* webpackChunkName: "components_global/icons/IconMenu" */).then(c => c.default || c),
  IconMoon: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\IconMoon.vue' /* webpackChunkName: "components_global/icons/IconMoon" */).then(c => c.default || c),
  IconSearch: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\IconSearch.vue' /* webpackChunkName: "components_global/icons/IconSearch" */).then(c => c.default || c),
  IconSun: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\IconSun.vue' /* webpackChunkName: "components_global/icons/IconSun" */).then(c => c.default || c),
  IconTranslate: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\IconTranslate.vue' /* webpackChunkName: "components_global/icons/IconTranslate" */).then(c => c.default || c),
  IconTwitter: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\IconTwitter.vue' /* webpackChunkName: "components_global/icons/IconTwitter" */).then(c => c.default || c),
  IconX: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\IconX.vue' /* webpackChunkName: "components_global/icons/IconX" */).then(c => c.default || c),
  IconXCircle: () => import('../..\\node_modules\\@nuxt\\content-theme-docs\\src\\components\\global\\icons\\IconXCircle.vue' /* webpackChunkName: "components_global/icons/IconXCircle" */).then(c => c.default || c),
  Combos: () => import('../..\\components\\global\\Combos.vue' /* webpackChunkName: "C:_repos/content-docs/components/global/Combos" */).then(c => c.default || c),
  DataSync: () => import('../..\\components\\global\\DataSync.vue' /* webpackChunkName: "C:_repos/content-docs/components/global/DataSync" */).then(c => c.default || c),
  OverrideContainerWidth: () => import('../..\\components\\global\\OverrideContainerWidth.vue' /* webpackChunkName: "C:_repos/content-docs/components/global/OverrideContainerWidth" */).then(c => c.default || c),
  SyncEntity: () => import('../..\\components\\global\\SyncEntity.vue' /* webpackChunkName: "C:_repos/content-docs/components/global/SyncEntity" */).then(c => c.default || c),
  SyncList: () => import('../..\\components\\global\\SyncList.vue' /* webpackChunkName: "C:_repos/configurator-docs/components/global/SyncList" */).then(c => c.default || c)
}

for (const name in globalComponents) {
  Vue.component(name, globalComponents[name])
}

```

The componets in the app gett this strange path name which also is seen in the error.

**Comment by pi0** [comment]
Issue is fixed with [components@v1.2.4](https://github.com/nuxt/components/releases/tag/v1.2.4). You can get update with `yarn upgrade nuxt` or `npm up nuxt`

**Comment by vuchl** [comment]
Hey @pi0 and @Atinux !

Thanks for helping out guys. It seems to fix the `generate` and `build` command!

Regards from snowy Saxony rocking the Nuxt merch!

![image](https://user-images.githubusercontent.com/4921147/104421912-537def80-557c-11eb-97e8-74b2915b8bb2.png)
